doc/api/util.md

```diff
@@ -934,8 +934,8 @@ is not supported.
 <!-- YAML
 added: v8.3.0
 changes:
-  - version: REPLACEME
-    pr-url: REPLACEME
```

Comment:
This should be:  https://github.com/nodejs/node/pull/22281

Reply:
ugh... we need to update the docs to request that people not use the `REPLACEME` as a token in the `pr-url` field. That needs to be filled in either before landing the PR or as it is landed so it doesn't get missed.

Reply:
This should have a lint rule, probably added to the mythical commit-queue, and in the meantime to ncu (@joyeecheung ?)